tests/test_PanOScomp.py

```diff
@@ -0,0 +1,54 @@
+import pytest
```

Comment:
You're close on this one, but it needs some redesign.  Assuming you've fixed the bug I noticed from before, these tests will not catch any future regressions introduced to the PanOSVersion comparison code.

So, there are six comparisons, and thus six comparison functions you've developed:
1) ==
2) !=
3) \>
4) \>=
5) \<
6) \<=

Thus you need six different test sets to verify everything, one `def` per comparison operation under test.  What I would like to see is that your six different test `def`s would take in three arguments:  two strings to turn into `PanOSVersion`s and whether the comparison under test should succeed or fail.